airbyte-integrations/connectors/destination-gcs/src/main/java/io/airbyte/integrations/destination/gcs/avro/GcsAvroWriter.java

```diff
@@ -85,4 +85,9 @@ protected void closeWhenFail() throws IOException {
     uploadManager.abort();
   }
 
+  @Override
+  public String getObjectPath() {
+    // TODO
+    throw new UnsupportedOperationException("not yet implemented");
```

Comment:
> S3Writer#getObjectKey is unimplemented in most classes, because I didn't want to dig into all of their behaviors.

Nitpick. Fairly enough. However, the next person who needs to change these classes may even have less context about what this method is about. I think this is relatively straightforward to implement them in this PR. The object path here is the `objectKey` in the constructor, which needs to be stored as an instance variable.

```suggestion
    return objectKey;
```

Reply:
I was more thinking about writing the unit tests. But it's probably better to at least have the implementation and worry about tests later; will add these changes in.